game-core/src/main/java/games/strategy/engine/data/PlayerList.java

```diff
@@ -55,15 +56,15 @@ public PlayerID getPlayerId(final String name) {
   }
 
   public Collection<String> getPlayersThatMayBeDisabled() {
-    return m_players.values().stream()
+    return players.values().stream()
         .filter(PlayerID::getCanBeDisabled)
         .filter(p -> !p.getIsDisabled()).map(DefaultNamed::getName)
```

Comment:
This looks like a formatting issue to me

Reply:
ebc13dd50